index.js

```diff
@@ -52,10 +72,31 @@ Use the higher-order function getWinners to do the following:
 💡 HINT: Don't worry about ties for now (Please see the README file for info on ties for a stretch goal.)
 4. Returns the names of all winning countries in an array called `winners` */ 
 
-function getWinners(/* code here */) {
-    /* code here */
+// //Without using .map(), .filter(), .forEach()
+// function getWinners(array, cbFinals) {
+//     const winners = []
+//     const finals = cbFinals(array)
+    
+//     for(let i=0; i < finals.length; i++){
+//         if(finals[i]['Home Team Goals'] > finals[i]['Away Team Goals']){
+//             winners.push(finals[i]['Home Team Name'])
+//        } 
+//        else {
+//            winners.push(finals[i]['Away Team Name'])
+//         }
+//    }
+//    return winners
+// }
+
+// console.log('Finals Winner:', getWinners(fifaData, getFinals))
+
+function getWinners(array, cbfinals){
+    const winners = cbfinals(array).map(match => {return match['Home Team Goals'] > match['Away Team Goals'] ? match['Home Team Name'] : match['Away Team Name']
```

Comment:
Nice job of using .map with a ternary expression! Also like how you figured it out without using array functions above it and commented it out to get a more lean code.